HomeTask2905/BankAcc.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HomeTask2905
+{
+    public class BankAcc
+    {
+        public double Balance { get; set; }
+
+        public string Owner { get; set; }
+
+        public BankAcc(double balance, string owner)
+        {
+            this.Balance = balance;
+            this.Owner = owner;
+        }
+
+        public void CurrentBalance()
```

Comment:
Incorrect name of a method.
Also since you added a public property, no need of this method.
Also please move all console to handler class.